frontend/src/ui/Blocks.tsx

```diff
@@ -0,0 +1,92 @@
+import React from "react";
+import { graphql, useFragment } from "react-relay/hooks";
+import type {
```

Comment:
```suggestion
import {
```

While `import type` is not wrong here, it's overly specific, IMO, and also unnecessary in this case. If we want to be this explicit about it, we should also be consistent, I think, which we aren't, right now.

For the record, `import type` is useful if you want to import things like `class`-es, which also exist at runtime, but you expressly only want them at compile time. `type`-s only exist at compile time anyway, though.

See also https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-8.html#type-only-imports-and-export.

Note, however, that `import type` might sometimes even be necessary because we use Babel. I created #146 to get a warning when we run into this.